lib/spectacles.rb

```diff
@@ -40,4 +40,13 @@ def trailer(stream)
   end
 end
 
+ActiveRecord::ConnectionAdapters::SchemaStatements.class_eval do
+  alias_method(:_spectacle_drop_table, :drop_table) if method_defined?(:drop_table)
+
+  # Add this so that views get cleaned up properly
+  def drop_table(name, options={})
+    execute "DROP TABLE #{name} CASCADE"
```

Comment:
Ah, I missed that. In that case, perhaps we should support them here? Also, I think it would be advantageous to use the `quote_table_name` helper for consistency.